src/pages/IstioRuleDetails/IstioRuleInfo.tsx

```diff
@@ -52,16 +72,47 @@ class IstioRuleInfo extends React.Component<RuleId, RuleInfoState> {
       });
   }
 
-  render() {
-    let actionList: any = [];
-    for (let i = 0; i < this.state.actions.length; i++) {
-      actionList.push(
-        <Col key={'ruleAction' + i}>
-          <IstioRuleInfoAction action={this.state.actions[i]} />
-        </Col>
-      );
+  parseSearch(): ParsedSearch {
+    if (this.props.search) {
```

Comment:
Can you add a comment about the expected string in this.props.search?

Reply:
This helper is more a private method to the class where we know the expected parameters.
In absence of them, the default behaviour is correct.
I'll add a comment.